### PR TITLE
replace onclick to use href

### PR DESCRIPTION
**Pull request by jomaora**
**Detailed purpose of the PR**

Don't user onClick function, but href. The internal urls on app-backoffice will dispatch the HISTORY actions without needing to pass a function. A <a> tag was added on the quickaccesscard component.

**Result and observation**

No changes on render.

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/molecule/quick-access-card/index.js [hunk 1]
```diff
@@ -77,17 +77,17 @@ const getBackgroudRadialColors = feature => {
 };
 
 const QuickAccessCard = (props, context) => {
-  const {title, description, feature, onClick, _blank} = props;
+  const {title, description, feature, href, _blank} = props;
```
Comment (DjamelSoualmi) [inline packages/@coorpacademy-components/src/molecule/quick-access-card/index.js:80]:
```suggestion
  const {title, description, feature, href, _blank = '_self'} = props;
```

File: packages/@coorpacademy-components/src/molecule/quick-access-card/index.js [hunk 2]
```diff
@@ -77,17 +77,17 @@ const getBackgroudRadialColors = feature => {
...
-  const handleClick = useMemo(() => e => onClick(e), [onClick]);
   const newTabOnClick = _blank ? <NewTabIcon className={style.newTabIcon} /> : null;
 
   return (
-    <div data-name="card" data-type={feature} className={style.quickAccess} onClick={handleClick}>
-      <div
-        style={{
-          background: `radial-gradient(62.12% 56.45% at 0% 77.29%, 
+    <div data-name="card" data-type={feature} className={style.quickAccess}>
+      <a href={href} target={_blank ? '_blank' : '_self'} className={style.link}>
```
Comment (DjamelSoualmi) [inline packages/@coorpacademy-components/src/molecule/quick-access-card/index.js:87]:
```suggestion
      <a href={href} target={_blank} className={style.link}>
```

File: packages/@coorpacademy-components/src/molecule/quick-access-card/index.js [hunk 3]
```diff
@@ -77,17 +77,17 @@ const getBackgroudRadialColors = feature => {
...
   const Icon = getIcon(feature);
-  const handleClick = useMemo(() => e => onClick(e), [onClick]);
   const newTabOnClick = _blank ? <NewTabIcon className={style.newTabIcon} /> : null;
 
   return (
-    <div data-name="card" data-type={feature} className={style.quickAccess} onClick={handleClick}>
-      <div
-        style={{
-          background: `radial-gradient(62.12% 56.45% at 0% 77.29%, 
+    <div data-name="card" data-type={feature} className={style.quickAccess}>
```
Comment (DjamelSoualmi) [inline packages/@coorpacademy-components/src/molecule/quick-access-card/index.js:86]:
```suggestion
    <div data-name="quick-access-card" data-type={feature} className={style.quickAccess}>
```
pour les tests d’intégration....

File: packages/@coorpacademy-components/src/molecule/quick-access-card/index.js [hunk 4]
```diff
@@ -97,46 +97,47 @@ const QuickAccessCard = (props, context) => {
...
     </div>
   );
 };
 
 QuickAccessCard.propTypes = {
   title: PropTypes.string,
   description: PropTypes.string,
-  onClick: PropTypes.func,
+  href: PropTypes.string,
   _blank: PropTypes.bool,
```
Comment (DjamelSoualmi) [inline packages/@coorpacademy-components/src/molecule/quick-access-card/index.js:141]:
```suggestion
  _blank: PropTypes.string,
```

**Review by DjamelSoualmi — CHANGES_REQUESTED** [review]

**Review by esa-coorp — APPROVED** [review]

**Pull request merged**